### ISSUE TITLE
Promote camera plugin to 1.0

### ISSUE THREAD

**Issue by jmagman** [flutter/flutter#151988]
[The camera package](https://pub.dev/packages/camera) is currently at 0.11.0+1.  It seems pretty mature at this point.  Can it be promoted to 1.0?

cc @stuartmorgan

**Comment by stuartmorgan** [comment]
The last time we did promotion to 1.0 we did a sprint of fixing top issues with the packages so it wasn't just arbitrary. We would ideally want to try to tackle a few top issues like https://github.com/flutter/flutter/issues/15953 as part of a 1.0 bump. We would also want to audit to make sure we didn't have any pending breaking changes we wanted to make.

Is there a specific driver for this, or just because it's one of the few we haven't done yet?

**Comment by jmagman** [comment]
> Is there a specific driver for this, or just because it's one of the few we haven't done yet?

Nope, I was just wondering if there was some particular feature or bug that was blocking it.  Nothing is pushing on this.  Feel free to close.

**Comment by stuartmorgan** [comment]
I think it's useful to use this as a place to start figuring out what checklist to 1.0 would look like; `camera` is probably overdue, and looking at 1.0 promotions had fallen off the radar during PM changes.
